mikeio1d/res1d.py

```diff
@@ -109,19 +124,21 @@ def __repr__(self):
 
             out.append(f"# Globals: {self.data.GlobalData.DataItems.Count}")
             for i, quantity in enumerate(self.data.Quantities):
-                out.append(f"{i} - {quantity.Id} <{quantity.EumQuantity.UnitAbbreviation}>")
+                out.append(
+                    f"{i} - {quantity.Id} <{quantity.EumQuantity.UnitAbbreviation}>"
+                )
 
         return str.join("\n", out)
 
-    #region Private methods
```

Comment:
This supposed to be a tag #region to depict regions as in PyCharm, etc. The concept of regions is borrowed from C# and maybe not fully "pythonic", but I like to be able to condense things like that. Now it is broken :-). We could go ahead and just get rid of these then, if we want the standard black behaviour. What do you think?

Reply:
OK, let's keep it like this :-). I found I can modify Sublime Text plugin settings to work with the space.